### ISSUE TITLE
No error message in console if .Net is not installed

### ISSUE THREAD

**Issue by LeakingAmps** [godotengine/godot#69034]
### Godot version

4.0 beta 4

### System information

Windows 10, AMD Ryzen 2400G

### Issue description

I attempted to load a project I had made in Godot Standard, in Godot C++, and because I had not installed the .net framework, the program just closed upon loading the project. Upon relaunching it with the console version, I discovered that there seems to be no error message of any kind for users not having the expected .net, or not having it in the right location.

### Steps to reproduce

Make a project in Godot standard
Save it
Close Godot standard
Open Godot C++ with the bat file [while .net 6 is not installed]
Attempt to open the project
Program crashes
No error in console

### Minimal reproduction project

[Game Files.zip](https://github.com/godotengine/godot/files/10072185/Game.Files.zip)

This is a cut-down copy of the particular project it happened with, but it should happen with any projects that you try to load a project in Godot C++ without .net

**Comment by Chaosus** [comment]
cc @neikeq

**Comment by Zireael07** [comment]
c++ is likely a typo for C#, isn't it?

**Comment by LeakingAmps** [comment]
I wrote this some time ago, but I believe so; if I remember correctly, this was because somewhere on the download chain, something listed it as the CPP version instead of C#

**Comment by raulsntos** [comment]
If .NET is not installed, Godot should be printing an error in the console. Maybe you just missed it, it should look something like this:

```
ERROR: The host fxr folder does not exist: /usr/share/dotnet/host/fxr
   at: try_get_path_from_dotnet_root (modules/mono/editor/hostfxr_resolver.cpp:323)
ERROR: .NET: One of the dependent libraries is missing. Typically when the `hostfxr`, `hostpolicy` or `coreclr` dynamic libraries are not present in the expected locations.
   at: find_hostfxr (modules/mono/mono_gd/gd_mono.cpp:126)
ERROR: .NET: Failed to load hostfxr
   at: initialize (modules/mono/mono_gd/gd_mono.cpp:397)
```

This will likely change with _Editor unification_ since this means there will be a single build for all users and we'll avoid crashing the editor when the .NET SDK is not found, instead showing a dialog and disabling .NET functionality.

**Comment by LeakingAmps** [comment]
I don't remember seeing that in console - if I remember correctly, it just stopped displaying new lines upon trying to load the file, then closed. I can test it again if necessary (uninstall .net and try again), but not sure if it's necessary given the second half of your post implies that the text or lack thereof will not be an issue in 4.0 or 4.1 release?

**Comment by akien-mga** [comment]
Fixed by #73815.
